src/legacy/ui/public/vis/components/tooltip/tooltip.js

```diff
@@ -22,9 +22,14 @@ import _ from 'lodash';
 import { Binder } from '../../../binder';
 import { positionTooltip } from './position_tooltip';
 import $ from 'jquery';
+import theme from '@elastic/eui/dist/eui_theme_light.json';
 
 let allContents = [];
 
+const tooltipColumnPadding = Number(theme.euiSizeXS.split('px')[0]) * 2;
```

Comment:
I think `parseInt` is a cleaner way to do this, no?

```suggestion
const tooltipColumnPadding = parseInt(theme.euiSizeXS || 0, 10) * 2;
```

Reply:
done in 34c2582